rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -136,20 +134,6 @@ public class Coordinator extends Application {
     private final LRAService lraService;
     private final RecoveryCoordinator recoveryCoordinator;
 
-    @Override
-    public Set<Class<?>> getClasses() {
-        HashSet<Class<?>> classes = new HashSet<>();
-        classes.add(Coordinator.class);
-        return classes;
-    }
-
-    @Override
-    public Set<Object> getSingletons() {
-        Set<Object> containerFilters = new HashSet<>();
-        containerFilters.add(new CoordinatorContainerFilter());
-        return containerFilters;
```

Comment:
Thanks @xstefank for the PR, you safely removed this because the auto discovery see the @Provider annotation in CoordinatorContainerFilter. Am I right?

Reply:
Yes, we also specify them in the service file.

Reply:
Perfect, thanks!